pact-support.gemspec

```diff
@@ -1,39 +1,42 @@
-# -*- encoding: utf-8 -*-
-lib = File.expand_path('../lib', __FILE__)
+# frozen_string_literal: true
+
+lib = File.expand_path("lib", __dir__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'pact/support/version'
+require "pact/support/version"
+
+Gem::Specification.new do |spec|
+  spec.name          = "pact-support"
+  spec.version       = Pact::Support::VERSION
+  spec.authors       = ["James Fraser", "Sergei Matheson", "Brent Snook", "Ronald Holshausen", "Beth Skurrie"]
+  spec.email         = ["james.fraser@alumni.swinburne.edu", "sergei.matheson@gmail.com", "brent@fuglylogic.com", "uglyog@gmail.com", "bskurrie@dius.com.au"]
 
-Gem::Specification.new do |gem|
-  gem.name          = "pact-support"
-  gem.version       = Pact::Support::VERSION
-  gem.authors       = ["James Fraser", "Sergei Matheson", "Brent Snook", "Ronald Holshausen", "Beth Skurrie"]
-  gem.email         = ["james.fraser@alumni.swinburne.edu", "sergei.matheson@gmail.com", "brent@fuglylogic.com", "uglyog@gmail.com", "bskurrie@dius.com.au"]
-  gem.summary       = %q{Shared code for Pact gems}
-  gem.homepage      = "https://github.com/bethesque/pact-support"
+  spec.summary       = "Shared code for Pact gems"
+  spec.homepage      = "https://github.com/pact-foundation/pact-support"
+  spec.license       = "MIT"
 
-  gem.required_ruby_version = '>= 2.0'
+  spec.required_ruby_version = ">= 2.0"
 
-  gem.files         = `git ls-files`.split($/)
-  gem.executables   = gem.files.grep(%r{^bin/}).map{ |f| File.basename(f) }
-  gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
-  gem.require_paths = ["lib"]
-  gem.license       = 'MIT'
+  spec.files         = Dir.chdir(File.expand_path(__dir__)) do
```

Comment:
This is stolen from auto-generated code from `bundle gem` but it looks a bit convoluted. Can we safely just include the following instead?

```ruby
  s.files = `git ls-files lib LICENSE.txt README.md`
            .split($RS)
```

Reply:
Can you try each one and just make sure there are no significant differences?

Reply:
@bethesque These are the results after I build and unpack gem locally

```bash
gem build pact-support.gemspec
gem unpack pact-support-1.11.0.gem
```

File diffs
```bash
# pact-support-1.11.0-old
# `git ls-files`.split($/)

# pact-support-1.11.0-new
# `git ls-files lib LICENSE.txt README.md`

$ diff -s pact-support-1.11.0-old/ pact-support-1.11.0-new/
Only in pact-support-1.11.0-old/: .github
Only in pact-support-1.11.0-old/: .gitignore
Only in pact-support-1.11.0-old/: .rspec
Only in pact-support-1.11.0-old/: .travis.yml
Only in pact-support-1.11.0-old/: Appraisals
Only in pact-support-1.11.0-old/: CHANGELOG.md
Only in pact-support-1.11.0-old/: Gemfile
Files pact-support-1.11.0-old/LICENSE.txt and pact-support-1.11.0-new/LICENSE.txt are identical
Files pact-support-1.11.0-old/README.md and pact-support-1.11.0-new/README.md are identical
Only in pact-support-1.11.0-old/: RELEASING.md
Only in pact-support-1.11.0-old/: Rakefile
Only in pact-support-1.11.0-old/: gemfiles
Common subdirectories: pact-support-1.11.0-old/lib and pact-support-1.11.0-new/lib
Only in pact-support-1.11.0-old/: pact-support.gemspec
Only in pact-support-1.11.0-old/: script
Only in pact-support-1.11.0-old/: spec
Only in pact-support-1.11.0-old/: tasks
```

Gem size diffs
```
# Old: 32KB
# New: 72KB
```